Source/ProjectFirma.Web/Views/Invoice/InvoiceGridSpec.cs

```diff
@@ -53,7 +54,7 @@ public InvoiceGridSpec(Models.Person currentPerson, bool invoiceFileExistsOnAtLe
             var userHasEditPermissions = new InvoiceEditFeature().HasPermissionByPerson(currentPerson);
             if (userHasEditPermissions)
             {
-                Add(string.Empty, x => DhtmlxGridHtmlHelpers.MakeEditIconAsModalDialogLinkBootstrap(new ModalDialogForm(x.GetEditUrl(), $"Edit {ObjectNameSingular} - {x.InvoiceNumber}"),
+                Add(string.Empty, x => DhtmlxGridHtmlHelpers.MakeEditIconAsModalDialogLinkBootstrap(new ModalDialogForm(x.GetEditUrl(), $"Edit {ObjectNameSingular} - {x.InvoiceIdentifyingName}"),

```

Comment:
This should be Invoice Number. That is what is used in the new documents provided to us.
The Invoice Identifying Name is a field from the previous implementation of invoices and not 100% certain it will be used.

Reply:
But invoice number is nullable?